pkgs/development/tools/rust/cargo-leptos/default.nix

```diff
@@ -15,24 +14,20 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "cargo-leptos";
-  version = "0.2.2";
+  version = "0.2.5";
 
   src = fetchFromGitHub {
     owner = "leptos-rs";
     repo = pname;
-    rev = "e98b478b0c82af1469151eff30f4246b9af4a539";
-    hash = "sha256-7o/yQanBBSA+MmMiGCYSjar4hZ8TRZoPiUniF5ELzXU=";
+    rev = version;
+    hash = "sha256-veRhTruM+Nw2rerzXC/kpi2Jr8mMMBLqOM2YBCpFePU=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
   };
 
-  nativeBuildInputs = optionals (!isDarwin) [ pkg-config ];
-
-  buildInputs = optionals (!isDarwin) [
-    openssl
-  ] ++ optionals isDarwin [
+  buildInputs = optionals isDarwin [
```

Comment:
Looks like Mac wants us to add SystemConfiguration here next to Security

Reply:
I've added `SystemConfiguration` but since I don't have a Mac, I have no idea if it works. It builds fine on Linux.